### DISCUSSION TITLE
Is is possible to inject carts like in the desktop version?

### DISCUSSION THREAD

**Discussion by AlexMini2517** [rafaelvcaetano/melonDS-android#1134] [Q&A]
On Android, I was only able to boot the firmware, but I can't insert any carts inside it.
![image](https://user-images.githubusercontent.com/76444133/227004968-350fec48-46cb-4d7b-988d-1fe2fbbe89d9.png)

**Comment by rafaelvcaetano** [comment]
On Android this is not currently possible.

**Comment by AlexMini2517** [comment]
Will it ever be implemented? If yes, approximately when? Thanks!
